wasm/bindings/TranslationModelBindings.cpp

```diff
@@ -10,10 +10,30 @@
 
 using namespace emscripten;
 
-// Binding code
+val getByteArrayView(marian::bergamot::AlignedMemory& alignedMemory) {
+  return val(typed_memory_view(alignedMemory.size(), alignedMemory.as<char>()));
+}
+
+EMSCRIPTEN_BINDINGS(aligned_memory) {
+  class_<marian::bergamot::AlignedMemory>("AlignedMemory")
+    .constructor<std::size_t, std::size_t>()
+    .function("size", &marian::bergamot::AlignedMemory::size)
+	  .function("getByteArrayView", &getByteArrayView)
+    ;
+}
+
+TranslationModel* TranslationModelFactory(const std::string &config,
+                                          marian::bergamot::AlignedMemory* modelMemory,
+                                          marian::bergamot::AlignedMemory* shortlistMemory) {
+  if (modelMemory == nullptr || shortlistMemory == nullptr) {
+    return nullptr;
```

Comment:
Do you ever check for that nullptr down the line? You should have a graceful failure

Reply:
We don't expect these to be nullptr. I kept it just to be cautious. I will remove it from here. 👍